portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -45,6 +45,22 @@ definition {
 			value1 = "Displaying Form: ${formName} (Modified)");
 	}
 
+	macro selectFormOnAPage {
```

Comment:
This has already been sent in [this](https://github.com/brianchandotcom/liferay-portal/pull/140723/files#diff-fc553d30c1bbfec555d47d6d690873e83a7545be1560d75ab1cde5269d371572R48) pr. As the sync hasn't been done yet, I had to cherry pick and update